README.md

```diff
@@ -1 +1,111 @@
-# covid-tracker
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Covid Tracker
+
+## About
+
+This is a covid tracker app where you can find latest information about COVID in Brazil, total number of cases, today new confirmed cases and deaths and more, you can also search by state and check specific information about any state of Brazil.
+
+Our development flow included: performing git flow practices & using components with routers, life cycle methods,handling events and state management by Redux store,actions, reducers in [duck Patern](https://github.com/erikras/ducks-modular-redux) with unit and component tests.
```

Comment:
- [ ] `Our development...` here gives the impression that this is a group project whereas it is a solo project. Kindly fix this.